packages/go/cypher/model/pgsql/translate/README.md

```diff
@@ -0,0 +1,50 @@
+# Translation
+
+# Graph Query Intermediate Representation
+
+## Intent
+
+The GQIR is a simplification of the openCypher query model. The model is designed to allow for a single-walk translation
+of a given openCypher query. The GQIR is used to normalize and reorganize query components to serialize them for
+encoding into different end-query models.
+
+### Pattern Rewriting
+
+The example below demonstrates reading out label constraints and reorganizing them to simplify translation to PgSQL:
```

Comment:
its gonna be really nice when this readme is fleshed out.  is it possible to mention here which components of the Translator manage each of these responsibilities?  ie. what structure manages pattern rewriting,identifier rebinding, etc.